google/oauth2/credentials.py

```diff
@@ -202,6 +218,7 @@ def from_authorized_user_info(cls, info, scopes=None):
             scopes=scopes,
             client_id=info["client_id"],
             client_secret=info["client_secret"],
+            quota_project=info.get("quota_project"),  # quota project may not exist
```

Comment:
we changed it to `quota_project_id` sorry

Reply:
Updated!